docs/installation/mtls.md

```
@@ -22,14 +22,34 @@ You'll need to place the following files in a directory on the host machine for
+
+**Docker Swarm:** add the directory to the `volumes` section for each `service` in the [Control Plane stack definition file](./installation.md#creating-the-stack-definition-file) and set these variables in the `environment` section:
 
 - `PGEDGE_HTTP__CA_CERT`: the path to the CA certificate
 - `PGEDGE_HTTP__SERVER_CERT`: the path to the server certificate
 - `PGEDGE_HTTP__SERVER_KEY`: the path to the server key
 - `PGEDGE_HTTP__CLIENT_CERT`: the path to the client certificate
 - `PGEDGE_HTTP__CLIENT_KEY`: the path to the client key
 
+**systemd:** place the certificate files on the host, then configure the paths in `/etc/pgedge-control-plane/config.json`:
+
+```json
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)